pkgs/by-name/cs/csvlens/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "csvlens";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "YS-L";
+    repo = "csvlens";
+    rev = "refs/tags/v${version}";
```

Comment:
It seems like dropping `refs/tags` also works here. Was wondering if there's any consensus about using one over the other.

Reply:
Some packages have branch names that look like tags.
For example, `v0.6` as a development branch and `v0.6` as a release tag.
I like to explicitly add `refs/tags/` because these are confusing.